presentation/placeholder-add-table/main.go

```diff
@@ -0,0 +1,83 @@
+// Copyright UniDoc ehf. All rights reserved.
```

Comment:
Do we need to add year in copyright?

Reply:
Updated at 8379fee68c74ec161e445c4d6fad986a6d8be406